kernelci/cli/base_api.py

```diff
@@ -63,12 +64,36 @@ class AttributesCommand(APICommand):
         {
             'name': 'attributes',
             'nargs': '*',
-            'help': "Attributes in name=value format",
+            'help': "Attributes in 'name=value' format, where = is "
+            "OPERATOR and can be one of: >, <, >=, <=, =",
         },
     ]
 
     @classmethod
     def _split_attributes(cls, attributes):
-        return dict(
-            tuple(attr.split('=')) for attr in attributes
-        ) if attributes else {}
+        rval = {}
```

Comment:
`rval` is not very meaningful, I first thought it was an "r-value" but now I guess you mean it's the returned value?  Maybe `ret` is less ambiguous here.  Still as per my other comment, I think there could be a slightly cleaner implementation as this function is now doing arguably more than one thing.  But that's not really a blocking issue as far as I can tell.

Reply:
Updated to ret